src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeGeneration/CSharpSyntaxTokens.cs

```diff
@@ -32,7 +32,9 @@ internal static class CSharpSyntaxTokens
     public static readonly SyntaxToken EndOfDocumentationCommentToken = Token(SyntaxKind.EndOfDocumentationCommentToken);
     public static readonly SyntaxToken EqualsToken = Token(SyntaxKind.EqualsToken);
     public static readonly SyntaxToken ExplicitKeyword = Token(SyntaxKind.ExplicitKeyword);
+#if !ROSLYN_4_12_OR_LOWER
     public static readonly SyntaxToken ExtensionKeyword = Token(SyntaxKind.ExtensionKeyword);
+#endif
```

Comment:
will be  used in RoslynAnalyzers, which will only be able to reference  up to Ros 4.12 (we're currently at ros 4.14).